src/MentionsTextInput.js

```diff
@@ -535,6 +535,17 @@ export default class MentionsTextInput extends Component {
       return;
     }
 
+    const isTextDifferenceGreaterThanOne = !this.state.text && text.length > 1 || this.state.text.length < text.length - 1;
```

Comment:
This would probably do well in it's on function too, right? It doesn't seem to have much effect on the rest of this function, and just needs the text value?

Reply:
ah theres already a getTextDifference function. the difference is that this one checks the difference before the the state is changed

Reply:
Yeah, but inline means you can't write a test for it, which I think is worthwhile, right?

Reply:
I think it'd be worth putting this logic inside of its own function, too

Reply:
👍